exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -192,6 +193,32 @@ apiVersion: v1`,
 				UID:        testCluster.UID,
 			})
 		}, timeout).Should(BeTrue())
-	})
 
+		// Wait until ClusterResourceSetBinding is created for the Cluster
+		var binding *addonsv1.ClusterResourceSetBinding
+		isBindingCreated := false
+		clusterResourceSetBindingKey := client.ObjectKey{
+			Namespace: testCluster.Namespace,
+			Name:      testCluster.Name,
+		}
+		for isBindingCreated == false {
+			binding = &addonsv1.ClusterResourceSetBinding{}
+
+			err := testEnv.Get(ctx, clusterResourceSetBindingKey, binding)
+			if err == nil {
+				isBindingCreated = true
+			}
+		}
```

Comment:
Would it make more sense to use Eventually here?

Reply:
I didn't use Eventually() here because, I want this to be blocking, because the next Eventually() requires CRSBinding to exist.

Reply:
You can have 2 different Eventually blocks - 1 here and 1 below - can't you?